### DISCUSSION TITLE
UI lib, c-lib

### DISCUSSION THREAD

**Discussion by ajpanton** [CorentinTh/it-tools#1381] [Q&A]
What does this section do? I can't really find a guide anywhere. I see it briefly mentioned in some change logs, but nothing that's helpful.

When clicking on boxes there, the browser's console gives errors like
> Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/html". Strict MIME type checking is enforced for module scripts per HTML spec.
> TypeError: error loading dynamically imported module

I installed it as a docker container without anything weird, pretty much just the port mapping.

**Comment by dacagi** [comment]
I think I may have found some useful info about this. See #400 where the dev is discussing about creating custom components. Specifically, I see a mention that this UI lib section is meant as a preview gallery **while on dev mode**, so it makes sense it doesn't show any behavior on the running app. I guess this kind of resolves our doubts, I was confused too but now I understand and looking forward to clone the repo and test this in my dev env. I'm trying to learn Vue for a personal project so this might be very helpful.
